### PR TITLE
Fix merge issues that made filtering completion to nothing crash VS

### LINKED ISSUE #11559: IntelliSense crashes if you change filters to a type with no suggestions
**Version Used**: 
VS '15' Preview 2

**Steps to Reproduce**:
1. type something to bring up intellisense
2. filter by a type
3. change your filter to a different type that displays no items

**Expected Behavior**:
I see "No Suggestions Available" in the intellisense menu
**Actual Behavior**:
VS crashes

### PR DESCRIPTION
**Pull request by Pilchie**
Fixes #11559

### CODE REVIEW

**Comment by Pilchie** [comment]
Tag @dotnet/roslyn-ide @CyrusNajmabadi

**Comment by Pilchie** [comment]
test vsi please

**Comment by CyrusNajmabadi** [comment]
:+1:

**Comment by Pilchie** [comment]
retest windows_debug_unit32_prtest please
retest windows_debug_unit64_prtest please
retest vsi p3 please

**Comment by Pilchie** [comment]
retest windows_vsi_p3_open_prtest please

**Comment by rchande** [comment]
:+1:

File: src/EditorFeatures/Core/Implementation/IntelliSense/Completion/Controller.cs [hunk 1]
```diff
@@ -125,8 +125,8 @@ internal override void OnModelUpdated(Model modelOpt)
             else
             {
                 var selectedItem = modelOpt.SelectedItem;
-                var viewSpan = modelOpt.GetViewBufferSpan(selectedItem.Item.Span);
-                var triggerSpan = modelOpt.GetCurrentSpanInSnapshot(viewSpan, this.TextView.TextSnapshot)
+                var viewSpan = selectedItem == null ? (ViewTextSpan?)null : modelOpt.GetViewBufferSpan(selectedItem.Item.Span);
```
Comment (rchande) [inline src/EditorFeatures/Core/Implementation/IntelliSense/Completion/Controller.cs]: We don't have any tests where we turn off everything through filters?

Comment (Pilchie) [inline src/EditorFeatures/Core/Implementation/IntelliSense/Completion/Controller.cs]: I don't think we have any tests for the filtering stuff.  I blame @CyrusNajmabadi.

**Comment by jasonmalinowski** [comment]
Can we add some tests before merging this, since clearly this is a test hole? :-)

**Comment by jasonmalinowski** [comment]
(otherwise :+1:)

**Comment by Pilchie** [comment]
Merging (before tests) to hit merge dates.  Will work on tests tomorrow (see #11700)

**Pull request merged**